packages/gestalt/src/AvatarGroup.js

```diff
@@ -0,0 +1,265 @@
+// @flow strict
+import React, { useState, type Node } from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Avatar from './Avatar.js';
+import Box from './Box.js';
+import Flex from './Flex.js';
+import Text from './Text.js';
+import Icon from './Icon.js';
+import TapArea from './TapArea.js';
+import { CompositeZIndex, FixedZIndex } from './zIndex.js';
+import { useColorScheme } from './contexts/ColorScheme.js';
+import { type AbstractEventHandler } from './AbstractEventHandler.js';
+import styles from './AvatarGroup.css';
+
+type Size = 'xs' | 'sm' | 'md';
+
+type Role = 'link' | 'button' | null;
```

Comment:
Making this optional `?('link' | 'button')` would improve the DX, allowing `undefined` as well. `null` isn't indicating anything in particular here, just the absence of one of the two values we care about

Reply:
DONE